### DISCUSSION TITLE
Uncaught ReferenceError: __dirname is not defined

### DISCUSSION THREAD

**Discussion by Netizen-alt** [alex8088/electron-vite#346] [Q&A]
import { ipcRenderer } from 'electron'

I can't run ipcRenderer through vue's page, what do I do with errors like this?

**Comment by alex8088** [comment]
https://electron-vite.org/guide/troubleshooting#uncaught-error-module-xxx-has-been-externalized-for-browser-compatibility-or-uncaught-referenceerror-dirname-is-not-defined
